src/components/ListItem.js

```diff
@@ -10,6 +10,16 @@ const ListItem = ({ item, token }) => {
   const [isPurchased, setPurchased] = useState(false);
   let numberOfPurchases = item.number_purchases || 0;
   let hoursDiff = getDifferenceInHours(item.last_purchased);
+  let daysDiff = getDifferenceInDays(item.last_purchased);
+
+  let [className, purchaseNext] =
+    daysDiff > 2 * item.next_purchase
+      ? ['inactive', 'inactive']
+      : item.next_purchase < 8
+      ? ['soon', 'soon']
+      : item.next_purchase < 15
+      ? ['kind-of-soon', 'kind of soon']
+      : ['not-soon', 'not soon'];
 
```

Comment:
Very nice. I can see these values maybe needing to be used as constants as the application grows. Not at all necessary to do now, but in the future you may want to create a map or object constant that defines these strings somewhere so they can be easily re-used and changed. like 
`const ITEM_STATE = {
     INACTIVE: {CLASS_NAME: 'inactive', PUCHASE_NEXT: 'inactive'},
     ...etc.
}`
Then you could use it like `ITEM_STATE.INACTIVE.CLASS_NAME`, etc.
I'm using all-caps because that's a convention we have on my current team for defining constants like that, but you don't have to use that (screaming snake case, I think it's called 🤣 )

Reply:
My next band will be called Screaming Snake Case.